src/api/migrations/20190702140355-create-comment.js

```diff
@@ -0,0 +1,45 @@
+export default {
+  up: (queryInterface, Sequelize) => queryInterface.createTable('Comments', {
+    id: {
+      allowNull: false,
+      autoIncrement: true,
+      primaryKey: true,
+      type: Sequelize.INTEGER
+    },
+    articleSlug: {
+      type: Sequelize.STRING,
+      references: {
+        model: 'Articles',
+        key: 'slug'
+      },
+      onDelete: 'CASCADE',
+      onUpdate: 'CASCADE',
+      allowNull: false
+    },
+    userId: {
+      type: Sequelize.INTEGER,
+      references: {
+        model: 'Users',
+        key: 'id'
+      },
+      onDelete: 'CASCADE',
+      onUpdate: 'CASCADE',
+      allowNull: false
+    },
+    commentBody: {
+      type: Sequelize.STRING
+    },
+    iteration: {
```

Comment:
@alainburindi what does iteration stand for? Also, does your task include creating article comments?